From 09a74cb8df22dbacef8888e9cb431cf7aec70809 Mon Sep 17 00:00:00 2001
From: Jordan Williams <jordan@jwillikers.com>
Date: Mon, 22 Apr 2024 14:35:50 -0500
Subject: [PATCH 16/21] meson: Fix build by default and install behavior for
 build-pipesz option

Signed-off-by: Jordan Williams <jordan@jwillikers.com>
(cherry picked from commit 205a065ced57659cc629c65bf6353afe1cb40030)
---
 meson.build | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 9c713bb015..c4aec5db92 100644
--- a/meson.build
+++ b/meson.build
@@ -3035,14 +3035,15 @@ if not is_disabler(exe)
   bashcompletions += ['hardlink']
 endif
 
-opt = not get_option('build-pipesz').disabled()
+opt = get_option('build-pipesz').allowed()
 exe = executable(
   'pipesz',
   pipesz_sources,
   include_directories : includes,
   link_with : [lib_common],
   install_dir : usrbin_exec_dir,
-  install : true)
+  install : opt,
+  build_by_default : opt)
 if opt and not is_disabler(exe)
   exes += exe
   manadocs += ['misc-utils/pipesz.1.adoc']
-- 
2.44.0

